vendor/assets/javascripts/react-chosen.js

```diff
@@ -0,0 +1,69 @@
+(function(window, React, $) {
```

Comment:
Right, but we'd only have to copy and paste that if we want a special build of React (e.g., with test addons).  To just use the `ui_components` gem it should work fine without any adjustments.   The dummy app is currently the only place that needs a special build.